features/step_definitions/active_docs_steps.rb

```diff
@@ -41,6 +41,28 @@
   assert_equal 1, evaluate_script("$('.apidocs-param-tips.#{autocomplete}:visible').length")
 end
 
+Then(/^the swagger v3 autocomplete should work for "(.*?)" with "(.*?)"$/) do |input_name, autocomplete|
+  id = 'default' # Could be passed as arg
+  section_id = "#operations-tag-#{id}"
+
+  closed_section = find("#{section_id}[data-is-open='false']")
+  closed_section&.click
+
+  within section_id do
+    method_id = "operations-#{id}-get_"
+    closed_method = find(method_id)
+    closed_method&.click
+
+    within method_id do
+      input = find("[data-param-name='#{input_name}'] input")
+      click_on 'Try it out'
+      # TODO: do the actual assertion
```

Comment:
@josemigallas Is this done ?

Reply:
No, this particular test case is marked as @wip.